File: Client/client/src/app/models/Infraestructura/InfraestructuraSaveModel.ts
export class InfraestructuraSaveModel {
    constructor(
        public Item: number = 0,
        public InfraestructuraId: number = 0,
        public SucursalId: number = 0,
        public Estado: number = 0,
        public CodigoSistema: string = '',
        public CodigoInterno: string = '',
        public Descripcion: string = '',
        public TipoInfraestructuraId: number = 0,
        public InfraestructuraDimensionId: number = 0,
        public Aforo: number = 0,
        public PisoId: number = 0,
        public FechaRegistro: Date = new Date,
        public CodUsuario: string = '',
        public EstadoRegistro: boolean = false,
        public Action: number = 0,
        public Seleccion: boolean = false
    ) { }
}

File: Desk/SpaceRT/Space.ServiceConsumer/Model/InfraestructuraSaveModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Space.ServiceConsumer
{
    public class InfraestructuraSaveModel
    {

        public Int32 InfraestructuraId { get; set; }
        public Int32 SucursalId { get; set; }
        public Int16 Estado { get; set; }
        public String CodigoSistema { get; set; }
        public String CodigoInterno { get; set; }
        public String Descripcion { get; set; }
        public Int32 TipoInfraestructuraId { get; set; }
        public Int32 ClasificacionId { get; set; }
        public Int32 InfraestructuraDimensionId { get; set; }
        public Int32 Aforo { get; set; }
        public Int32 PisoId { get; set; }
        public DateTime FechaRegistro { get; set; }
        public String CodUsuario { get; set; }
        public Boolean EstadoRegistro { get; set; } = true;
    }
}
